internal/cmd/asyncapi/command_export.go

```diff
@@ -279,35 +272,16 @@ func (c command) getMessageExamples(consumer *ckgo.Consumer, topicName, contentT
 	return jsonMessage, nil
 }
 
-func (c *command) getBindings(cluster *ccstructs.KafkaCluster, topicDescription kafkarestv3.TopicData) (*bindings, error) {
-	kafkaREST, err := c.GetKafkaREST()
-	if err != nil {
-		return nil, err
-	}
-	configs, err := kafkaREST.CloudClient.ListKafkaTopicConfigs(cluster.GetId(), topicDescription.GetTopicName())
+func (c *command) getBindings(kafkaREST *pcmd.KafkaREST, clusterId string, topicName string) (*bindings, error) {
+	configs, err := kafkaREST.CloudClient.ListKafkaTopicConfigs(clusterId, topicName)
 	if err != nil {
 		return nil, err
 	}
-	var cleanupPolicy string
-	deleteRetentionMsValue := -1
+	configsMap := make(map[string]string)
 	for _, config := range configs.Data {
-		if config.Name == "cleanup.policy" {
-			cleanupPolicy = config.GetValue()
-		}
-		if config.Name == "delete.retention.ms" {
-			deleteRetentionMsValue, err = strconv.Atoi(config.GetValue())
-			if err != nil {
-				return nil, err
-			}
-		}
-	}
-	var channelBindings any = confluentBinding{
-		Configs: Configs{
-			CleanupPolicy:                  cleanupPolicy,
-			DeleteRetentionMs:              deleteRetentionMsValue,
-			ConfluentValueSchemaValidation: "true",
-		},
+		configsMap[config.Name] = config.GetValue()
```

Comment:
I assume that if there's a `GetValue()` there's also a `GetName()`?
```suggestion
		configsMap[config.GetName()] = config.GetValue()
```